CHANGELOG.md

```diff
@@ -6,15 +6,15 @@
 
 ### Breaking Changes
 
-_None_
+- Propose to retry when `gp_downloadmetadata` receives a `429 - Too Many Requests` error. [#406]
```

Comment:
I'm not sure this is really a breaking change — as in, that won't require any change to client repos adopting the new release-toolkit to update their call sites or config or whatnot, will it?

I think it's not worth a major version bump on the next release, while a New Feature and minor bump would make more sense for this change 🙃

Reply:
Yep yep. That was an oversight which I realized only after merging this. It's fixed in 39808dab6bace7ed77e403c36494422303127647.

Reply:
Oh, perfect, thanks for following-up 👍